releasenotes/notes/cleanup-curve-analysis-96d7ff706cae5b4e.yaml

```
@@ -0,0 +1,29 @@
+---
+upgrade:
+    :class:`BaseCurveAnalysis` class has been added as a superclass of :class:`CurveAnalysis`.
```

I think you also need to add `.` before all the classes methods to have them looked up and work as actual links to the API docs. Eg 
```suggestion
    :class:`.BaseCurveAnalysis` class has been added as a superclass of :class:`.CurveAnalysis`.
```
without it it will just render as a codeblock but not be clickable